content/how-it-works.md

```diff
@@ -0,0 +1,30 @@
+title: complate Internals and Adaptors
+h1: complate Internals and Adaptors
+subtitle: How it works
```

Comment:
terrible title(s)

Reply:
We could leave out the subtitle completely and just keep the `complate Internals and Adaptors` title. And then maybe the html URI would be something like `complate-adaptors.html` instead of `how-it-works.html`

Just a suggestion.

Reply:
I went with "complate Adaptors" 🤷‍♂